gateway/src/main/java/org/georchestra/gateway/security/oauth2/OpenIdConnectUserMapper.java

```diff
@@ -207,7 +207,6 @@ public class OpenIdConnectUserMapper extends OAuth2UserMapper {
                 } catch (DataServiceException e) {
                     throw new IllegalStateException(e);
                 } catch (Exception e) {
-                    throw new RuntimeException(e);
```

Comment:
Maybe log the exception instead of completely ignoring it ?

Reply:
ok

Reply:
PR closed @pmauduit . it has been cancelled, no need to it.